pagerduty/slack_connection.go

```diff
@@ -0,0 +1,123 @@
+package pagerduty
+
+import "fmt"
+
+// SlackConnectionService handles the communication with the integration slack
+// related methods of the PagerDuty API.
+type SlackConnectionService service
+
+// SlackConnection represents a slack connection.
+type SlackConnection struct {
+	ID               string           `json:"id,omitempty"`
+	SourceID         string           `json:"source_id,omitempty"`
+	SourceName       string           `json:"source_name,omitempty"`
+	SourceType       string           `json:"source_type,omitempty"`
+	ChannelID        string           `json:"channel_id,omitempty"`
+	ChannelName      string           `json:"channel_name,omitempty"`
+	Config           ConnectionConfig `json:"config,omitempty"`
+	NotificationType string           `json:"notification_type,omitempty"`
```

Comment:
I may just be skimming past it, but I'm not seeing `NotificationType` in the current API docs, or is it necessary for some other reason?

Reply:
You're right. That field isn't documented. I'll remove it.